src/games/entities.ts

```diff
@@ -1,8 +1,9 @@
 import { BaseEntity, PrimaryGeneratedColumn, Column, Entity, Index, OneToMany, ManyToOne } from 'typeorm'
 import User from '../users/entity'
 
-export type Row = [ string , string ,string ,string ,string ,string ,string ,string ,string ,string]
-export type Board = [ Row, Row, Row, Row, Row, Row,Row, Row, Row, Row]
```

Comment:
thanks for removing this! It's prettier now 😆

Reply:
Sure :+1: